api/helpers/utils.js

```diff
@@ -89,8 +89,87 @@ const updateJuvenileName = (name, parentArrayName = []) => {
   return updatedName;
 };
 
+const nyCities = ["New York", "New York City", "NYC"];
+const nyCounties = ["Richmond", "Queens", "New York", "Kings", "Bronx"];
+const nyStates = ["NY", "New York"];
+const listOfStates = [
```

Comment:
This is fine, but there are packages that would probably do a lot of this for you (maybe `https://www.npmjs.com/package/usa-states`), without having to manually list everything.

Reply:
I didn't want to bring in a package for this and this seemed the simplest solution.